dlt/common/runtime/exec_info.py

```diff
@@ -45,7 +45,11 @@ def exec_info_names() -> List[TExecInfoNames]:
         names.append("colab")
     if airflow_info():
         names.append("airflow")
-    return names
+    if is_aws_lambda():
+        names.append("aws_lambda")
+    if is_gcp_cloud_function():
+        names.append("gcp_cloud_function")
+        return names
```

Comment:
please fix indentation, now the function returns any data only if on cloud function

Reply:
Hi @rudolfix , indentation has been fixed. Please proceed.